app/src/main/java/org/coepi/android/ui/onboarding/OnboardingAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package org.coepi.android.ui.onboarding
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+import io.reactivex.subjects.Subject
+import org.coepi.android.R
+import org.coepi.android.ui.onboarding.OnboardingCard.LargeOnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingCard.SmallOnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingFragment.Companion.LARGE_CARD
+import org.coepi.android.ui.onboarding.OnboardingFragment.Companion.SMALL_CARD
+import org.coepi.android.ui.onboarding.binders.LargeOnboardingCardBinder
+import org.coepi.android.ui.onboarding.binders.SmallOnboardingCardBinder
+
+class OnboardingAdapter(private val data: List<OnboardingCard>) :
+    RecyclerView.Adapter<OnboardingViewHolder>() {
+
+    val onboardingEventStream: Observable<OnboardingClickEvent>
+        get() = onboardingEventSubject
+
+    private val onboardingEventSubject = PublishSubject.create<OnboardingClickEvent>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
+        OnboardingViewHolder(
+        when (viewType) {
+            SMALL_CARD -> LayoutInflater.from(parent.context).inflate(R.layout.card_onboarding_small, parent, false)
```

Comment:
You can use static import here (alt+enter or opt+enter and "add import...) both on `from` and the layout name, to make this a bit shorter.